docs/source/ruletypes.rst

```diff
@@ -510,6 +524,85 @@ This value is added in back of the event. For example,
 
 ``kibana4_end_timedelta: minutes: 2``
 
+use_kibana_discover
```

Comment:
Kind of thinking this should be renamed to `generate_kibana_discover_link`. 

Thoughts?

Reply:
This also makes me think that maybe the variable should be `kibana_discover_link` instead of `kibana_link`.  Not that I have a use case for it, but that would allow users to link to both dashboard and discover apps.